ietf/submit/tests.py

```diff
@@ -541,7 +555,7 @@ def test_submit_new_replaced_wg_as_author(self):
     def test_submit_new_wg_with_extresources(self):
         self.submit_new_draft_with_extresources(group=GroupFactory())
 
-    def submit_existing(self, formats, change_authors=True, group_type='wg', stream_type='ietf'):
+    def submit_existing(self, formats, author=None, email=None, change_authors=True, group_type='wg', stream_type='ietf'):
```

Comment:
These new parameters don't seem to be used in the method, or am I missing something?